docs/index.html

```diff
@@ -76,6 +76,7 @@
         rows[i].onclick = function () {
             var cssClass = this.getAttribute( 'class' );
             var iconID = cssClass.split(' gridicons-')[1];
+			alert(iconID);
```

Comment:
Running grunt again removes this alert. Let me know if you'd actually like this here and I can update the Gruntfile